vnext/Shared/MemoryMappedBuffer.cpp

```diff
@@ -0,0 +1,154 @@
+#include "pch.h"
+#include "MemoryMappedBuffer.h"
+
+#include <werapi.h>
+#include <windows.h>
+
+namespace {
+
+class MemoryMappedBuffer : public facebook::jsi::Buffer {
+ public:
+  MemoryMappedBuffer(const wchar_t *const filename, uint32_t offset);
+
+  size_t size() const override;
+  const uint8_t *data() const override;
+
+ private:
+  static void fileDataDeleter(void *p) {
```

Comment:
>void *p [](start = 30, length = 7)

(nit) "void* p" ?

Reply:
The "void *p" style is enforced through clang format.

---
In reply to: [320883993](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320883993) [](ancestors = 320883993)